CONTRIBUTING.md

```diff
@@ -96,6 +96,30 @@ package (PyQt5). This can be achieved by running:
   $ pip install pyqt5
 ```
 
+### Using the correct version of spyder-kernels
+
+Following the separation of ``spyder-kernels`` into its own modular package
+in Spyder 3.3.0, you'll need to have the corresponding version of that package
+available—``0.x`` for Spyder 3 (``3.x`` branch), and ``1.x`` for Spyder 4
+(``master`` branch). The above procedure will install the ``0.x`` version;
+to test the ``master`` branch (Spyder 4), you'll need to install the
+corresponding ``1.x`` version of ``spyder-kernels``.
+
+This can be done via two methods: installing the correct version via ``conda``
+(or ``pip``):
+
+```bash
+ conda install spyder-kernels=1.*
+```
+
+(and using ``conda install spyder-kernels=0.*`` to switch back to the
+Spyder 3 version), or by ``clone``-ing the `spyder-kernels git repository`_
```

Comment:
But this option (cloning the `spyder-kernels` repo) also requires that you run

    pip install -e .

on its root. Else Spyder won't find the package.

Reply:
Also, this

    `spyder-kernels git repository`_

is rst syntax. Please use markdown here.

Reply:
> But this option (cloning the spyder-kernels repo) also requires that you run

Thanks; added statement to that effect.

> rst syntax

Silly copy and paste omission that I forgot to fix; avoiding similar errors as much as much as possible is why I specify always using double backtick, single asterisk, etc in markdown.